SPECS/fmt/fmt.spec

```diff
@@ -0,0 +1,225 @@
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+%undefine __cmake_in_source_build
+
+%bcond_with doc
+Name:           fmt
+Version:        7.0.3
+Release:        3%{?dist}
+Summary:        Small, safe and fast formatting library for C++
+
+License:        BSD
+URL:            https://github.com/fmtlib/%{name}
+Source0:        %{url}/archive/%{version}.tar.gz
```

Comment:
Let's re-name the tarball to `%{name}-%{version}.tar.gz`. Using only the version number is prone to conflicts.